.github/workflows/devnet-deploys.yml

```diff
@@ -128,12 +141,6 @@ jobs:
         run: |
           terraform apply -input=false -auto-approve -replace="aws_efs_file_system.node_data_store"
 
-      - name: Deploy Faucet
```

Comment:
This was moved to a separate job in order to have access to all contract addresses